packages/design/src/icons/getIcon.ts

```diff
@@ -54,7 +54,8 @@ export function getIcon({ name, color, size = "base" }: IconProps) {
 
   const pathClassNames = getPathClassNames(name, color);
   const paths = getPaths(name);
-  const viewBox = `0 0 ${iconMap.height} ${iconMap.height}`;
+  const iconSize = name === "truck" ? 1024 : iconMap.height;
```

Comment:
SVG's are not the same as images. The paths for it are made in a 1024 canvas which makes sense since it has more intricate details. Its paths are much more complicated than our icons.

Reply:
ahh